hkmc2/shared/src/main/scala/hkmc2/codegen/StackSafeTransform.scala

```diff
@@ -121,7 +136,6 @@ class StackSafeTransform(depthLimit: Int, paths: HandlerPaths, doUnwindMap: Map[
       
     val doUnwindPath = doUnwindMap.get(fnOrCls)
     val newBody = transform(blk, curDepth)
-    
     if isTrivial(blk) then
       newBody
```

Comment:
If you remove this line, maybe also remove the one above. Or keep both for readability?